test/helper.rb

```diff
@@ -123,11 +123,25 @@ def with_phenix
     before do
       ActiveRecord::Base.stubs(:with_default_shard).yields
 
+      # Create intentionally empty databases
+      Phenix.configure do |config|
+        config.skip_database = lambda do |name, _|
+          intentionally_empty_databases = %w[test3 test3_shard_0]
+
+          !intentionally_empty_databases.include?(name)
+        end
+      end
+      Phenix.rise!(with_schema: false)
```

Comment:
This is actually not correct behaviour. For some cases where the `test3` env is used, we’ll probably need the DB schema.

I’m thinking populating *all* databases before every test probably isn’t a really good idea after all. Or perhaps the concept of switching between different `Rails.env`’s which have their database config in the same file is the idea that isn’t really good…